public/app/plugins/panel/geomap/layers/data/dayNightLayer.tsx

```diff
@@ -55,8 +56,9 @@ export const defaultDayNightConfig: MapLayerOptions<DayNightConfig> = {
 export const dayNightLayer: MapLayerRegistryItem<DayNightConfig> = {
   id: DAY_NIGHT_LAYER_ID,
   name: 'Night / Day',
-  description: 'Show daylight regions',
+  description: 'Show day and night regions',
   isBaseMap: false,
+  state: PluginState.alpha,
```

Comment:
Is the alpha label supposed to show up here?
![image](https://user-images.githubusercontent.com/60050885/174158192-8640eb25-1c51-47b9-8289-be845c9ed0ed.png)

Reply:
You are seeing that because you have alpha panels enabled... I just pushed this:  
<img width="462" alt="image" src="https://user-images.githubusercontent.com/705951/174161389-42b23777-9246-4dc0-8321-dd8f17547c3a.png">

not sure about the icon, but it does make them more noticeable!

Reply:
actually this makes them last now... a little less weird: 
<img width="428" alt="image" src="https://user-images.githubusercontent.com/705951/174161930-8bdb91c6-04dc-4114-9769-3ee3e776b67e.png">

Reply:
Looks great!